src/main/java/smells/incorrectorderofmodifiers/IncorrectOrderOfModifiers.java

```diff
@@ -10,7 +10,7 @@
 	/**
 	 * expected: private static final int
 	 */
-	private final static int MULTIPLIER = 10;
+	private static final int MULTIPLIER = 10;
```

Comment:
@Samantha-Bo Reorder modifiers!

Reply:
Could not exchange data with wit.ai! Please rephrase your instruction in a new comment.

Reply:
@Samantha-Bo please reorder modifiers :)

Reply:
All modifiers are in correct order! Nothing to refactor. Please rephrase your instruction in a new comment.